convert2rhel/utils/__init__.py

```diff
@@ -1115,3 +1116,20 @@ def write_json_object_to_file(path, data, mode=0o600):
     with open(path, mode="w") as handler:
         os.chmod(path, mode)
         json.dump(data, handler, indent=4)
+
+
+def warn_deprecated_env(env_name):
+    """Warn the user that the environment variable is deprecated.
+
+    :param env_name: The name of the environment variable that is deprecated.
+    :type env_name: str
+    """
+    if env_name not in os.environ:
+        # Nothing to do here.
+        return
```

Comment:
I'd rather be explicit. If it doesn't fit then we can change it to return 0 and 1 or something but I do want to have explicit returns
```suggestion
        return False
```

Or we can change it to be kinda like getting the environment variable but also warning
```suggestion
        return None
```